scripts/python/csv_merger.py

```diff
@@ -0,0 +1,62 @@
+import csv
+import glob
+from pathlib import Path
+import os
+from typing import List, Generator, Iterator, TextIO, Tuple
+import typer
```

Comment:
I am a bit wary of using external libraries in scripts. I'd rather stick with `argparse`.

Reply:
I understand the concern. But,  Imo, it shouldn't matter for the following reasons:

* We have been using it already
* It is a simple process (`pip install requirements`)
* Makes the code simpler and tidier
* At the end, we use some other deps (like `yaml`, that you will still need to install, so a few more should be ok)

Reply:
Well, I guess since we have `requirements.txt` I think it should be fine. Fair point about using other deps.